code/game/objects/items/tools/weldingtool.dm

```diff
@@ -322,10 +328,9 @@
 
 /obj/item/weldingtool/largetank/cyborg
 	name = "integrated welding tool"
-	desc = "An advanced welder designed to be used in robotic systems. Custom framework doubles the speed of welding."
+	desc = "An advanced welder designed to be used in robotic systems."
```

Comment:
isn't this used by engiborgs? The speed of the engiborg welder shouldn't be reduced.

Reply:
This was just a pure lie from what I can tell, so.

Reply:
![image](https://user-images.githubusercontent.com/10366817/189577267-19007940-5279-4f3b-8a06-e272b4e89622.png)

Reply:
Does that not do anything?